feflow/tests/test_protein_mutation.py

```diff
@@ -1,6 +1,187 @@
 """Test related to the protein mutation protocol and utility functions"""
 
+import json
+from importlib.resources import files
+from pathlib import Path
+
+import pytest
+from gufe import ProteinComponent, ChemicalSystem, ProtocolDAGResult
+from gufe.protocols.protocoldag import execute_DAG
+from feflow.protocols import ProteinMutationProtocol
+
+
+# Fixtures
+@pytest.fixture(scope="module")
+def ala_capped():
+    """ProteinComponent for Alanine residue capped by ACE and NME."""
+    input_pdb = str(files("feflow.tests.data").joinpath("ALA_capped.pdb"))
+    protein_comp = ProteinComponent.from_pdb_file(input_pdb)
+    return protein_comp
+
+
+@pytest.fixture(scope="module")
+def gly_capped():
+    """ProteinComponent for Glycine residue capped by ACE and NME"""
+    input_pdb = str(files("feflow.tests.data").joinpath("GLY_capped.pdb"))
+    protein_comp = ProteinComponent.from_pdb_file(input_pdb)
+    return protein_comp
+
+
+@pytest.fixture(scope="module")
+def asp_capped():
+    """ProteinComponent for Aspartic Acid residue capped by ACE and NME.
+    This is meant to be used for testing charge transformations."""
+    input_pdb = str(files("feflow.tests.data").joinpath("ASP_capped.pdb"))
+    protein_comp = ProteinComponent.from_pdb_file(input_pdb)
+    return protein_comp
+
+
+@pytest.fixture(scope="module")
+def leu_capped():
+    """ProteinComponent for Leucine residue capped by ACE and NME.
+    This is meant to be used for testing charge transformations."""
+    input_pdb = str(files("feflow.tests.data").joinpath("LEU_capped.pdb"))
+    protein_comp = ProteinComponent.from_pdb_file(input_pdb)
+    return protein_comp
+
```

Comment:
Do we not have a `black` precommit set up? We should!

Reply:
Oh, I think we lost that with the migration to the OpenFreeEnergy, yeah, we should probably bring that back.